### DISCUSSION TITLE
The ChatGPT-Writer isn't generating a response even after opening ChatGPT as well.

### DISCUSSION THREAD

**Discussion by MamiMrl** [GorvGoyl/ChatGPT-Writer#95] [Report Bugs and Issues]
The following message was always occurring when chatgpt wasn't open in the browser for long time:

"There was an unknown error. Make sure you are logged-in to [chat.openai.com](https://chat.openai.com/)
If issue persists, please [report it here](https://github.com/GorvGoyl/ChatGPT-Writer/discussions/categories/report-bugs-and-issues).
Tired of frequent issues? Upgrade to Pro to get seamless experience. Learn more"

When I click on the chat.openai.com link, it still persists and doesn't generate responses to the mail.

**Comment by sunilsinghco** [comment]
No working

**Comment by cancerian8245** [comment]
The same issue here, It's now a frustrating time.
